src/pattern_library/css/global/global-styles.css

```diff
@@ -0,0 +1,381 @@
+html {
+	background: var(--color-blue-100);
+}
+
+body {
+	color: var(--color-grey-800);
+	font-size: var(--type-step-0);
+	font-family: var(--font-base);
+	line-height: var(--leading-standard);
+	font-size-adjust: from-font;
+	margin: 0;
+	padding-top: calc(var(--navigation-height) + var(--gutter));
+}
+
+:is(h1, h2, h3) {
+	line-height: var(--leading-fine);
+	text-wrap: balance;
+	font-weight: var(--font-normal);
+}
+
+h1 {
+	font-size: var(--type-step-3);
+	max-width: 20ch;
+}
+
+h2 {
+	font-size: var(--type-step-2);
+	max-width: 35ch;
+}
+
+h3 {
+	font-size: var(--type-step-1);
+	max-width: 35ch;
+}
+
+:is(h4, h5, h6) {
+	font-size: var(--type-step-0);
+	font-weight: var(--font-normal);
+}
+
+small {
+	font-size: var(--type-step-00);
+}
+
+ins {
+	text-decoration: none;
+	background: var(--color-primary);
+	color: var(--color-dark);
+	padding-inline: 0.3em;
+}
+
+:is(code, kbd, samp) {
+	font-family: var(--font-mono);
+	padding: 0.2em 0.2em 0.05em 0.2em;
+	hyphens: none;
+	tab-size: 2;
+	text-align: left;
+	word-spacing: normal;
+	word-break: normal;
+	word-wrap: normal;
+	box-decoration-break: clone;
+}
+
+/* This is quite a new property, so we want code styles to at least, not be huge or tiny */
+@supports not (font-size-adjust: from-font) {
+	:is(code, kbd, samp) {
+		font-size: 0.8em;
+	}
+}
+
+pre {
+	width: max-content;
+	max-width: 100%;
+	overflow-x: auto;
+	background: var(--color-grey-100);
+	padding: var(--flow-space, 1em);
+	tab-size: 4ch;
+}
+
+pre code {
+	border: none;
+	background: none;
+	padding: 0;
+}
+
+kbd {
+	border: 1px solid;
+	padding-block-end: 0.1em;
+}
+
+var {
+	font-style: normal;
+	font-weight: var(--font-medium);
+}
+
+ul:not([class]) {
+	padding-inline-start: 1.7ch;
+	list-style-type: disc;
+}
+
+ul:not([class]) > li {
+	padding-inline-start: 0.9ch;
+}
+
+ol:not([class]) {
+	padding-inline-start: 2.1ch;
+}
+
+ol:not([class]) > li {
+	padding-inline-start: 0.5ch;
+}
+
+:is(ol, ul):not([class]) li + * {
+	margin-block-start: var(--flow-space, var(--space-2xs));
+}
+
+ul ::marker {
+	font-size: 0.8lh;
+}
+
+ol ::marker {
+	font-size: 1em;
+	font-weight: var(--font-bold);
+}
+
+dt {
+	font-weight: var(--font-medium);
```

Comment:
On my Linux machine (Firefox), I don't seem to have a "medium" version of the font. For me, the definition list example just looks like broken indentation and has no other visual indication of hierachy.

Reply:
I thought I set up those weights to match the current production styles but it might be something I pulled in from the course. But yeah we don't embed a medium version of the font, so it shoudl be font-normal